src/main/java/org/craftercms/engine/graphql/SchemaUtils.java

```diff
@@ -17,15 +17,10 @@
 
 package org.craftercms.engine.graphql;
 
-import java.util.Arrays;
-import java.util.List;
+import java.util.*;
 
-import graphql.schema.GraphQLArgument;
-import graphql.schema.GraphQLEnumType;
-import graphql.schema.GraphQLFieldDefinition;
-import graphql.schema.GraphQLInputObjectField;
-import graphql.schema.GraphQLInputObjectType;
-import graphql.schema.GraphQLObjectType;
+import graphql.schema.*;
```

Comment:
Please don't include with *

Reply:
That's IntelliJ's optimizer, when there are too many imports in that package it includes all, I would have to manually enter each one then.